### DISCUSSION TITLE
中華電信投入智慧醫療 高雄榮家智慧長照服務上線啟動

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#455] [General]
中華電信投入智慧醫療 高雄榮家智慧長照服務上線啟動
這種利用人工智慧照顧老年人的，在台灣會是一個很大的商機
https://www.technice.com.tw/ai/1590/

**Comment by BingRong1217** [comment]
越來越多企業投入AI人工智慧的發展
